docs/guides/graphics_tutorial.rst

```diff
@@ -0,0 +1,279 @@
+.. image:: images/graphics_tutorial/add_point_light_texture.png
+	:alt: Add Point Light
+	:align: center
+
+
+If you press 'q', a transform gizmo will appear on the object you have selected. You can adjust light location through it.
+For rotation and scaling gizmo, press 'w' and 'e', respectively.
+
+.. image:: images/graphics_tutorial/translate_point_light.png
+	:alt: Translate Point Light 1
+	:align: center
+.. image:: images/graphics_tutorial/translate_point_light_2.png
+	:alt: Translate Point Light 2
+	:align: center
+.. image:: images/graphics_tutorial/translate_point_light_3.png
+	:alt: Translate Point Light 3
+	:align: center
+
+
+*********************
+Overriding a material
+*********************
+
+To change the properties of one object without changing the original material, we can use overriding materials. To override a material:
+
+1) Create a new material by going to file -> Create Resource.
+2) Name the material and click create.
+
+.. image:: images/graphics_tutorial/create_basic_material.png
+	:alt: Create Basic Material
+	:align: center
+
+3) Go to the `inspector-> model component->Materials`.
+4) click on the folder icon on the corresponding overriding material.
+
+.. image:: images/graphics_tutorial/select_the_newly_created_basic_material.png
+	:alt: Select Basic Material
+	:align: center
+
+Now you can change its basic textures by 
+1)clicking on the pencil icon 
+2)In the file viewer now click on the diffuse texture and select the appropriate diffuse texture.
+
+.. image:: images/graphics_tutorial/change_basic_material_texture.png
+	:alt: Change Basic Material Texture
+	:align: center
+
+***************
+Custom Material
+***************
+
+
+
+1) Go to create Resource -> Custom Material.
+
+.. image:: images/graphics_tutorial/create_custom_material.png
+	:alt: Create Custom Material
+	:align: center
+
+2) Enter material name.
+3) Now go to Inspector -> ModelComponent and then to Materials.
+4) Click on the folder icon and choose the material.
+
+.. image:: images/graphics_tutorial/select_the_newly_created_custom_material.png
+	:alt: Select Custom Material
+	:align: center
+
+***************
+Adding a shader
+***************
+
+To Add shader:
+
+1) Click on the pencil icon on the overriding custom material.
+2) Now, in the file viewer you'll get options to add vertex and pixel shaders.
+3) Click on the pixel shader. A dialog box will open now you can just select the shader.
+
+.. image:: images/graphics_tutorial/change_custom_material_pixel_shader.png
+	:alt: Change Custom Material Pixel Shader
+	:align: center
+
+You can use fire_pixel_shader from rootex/core/renderer/shaders
+
+Clicking on the pencil icon opens an editor to customise the shader.
+
+**Note:** You can only add shaders to custom materials. If you want to use default material, override the original default material with custom material and then add a shader to the overriding material. The overriding material does inherit the textures of the original materials.
```

Comment:
You can use use a box for writing notes.
Refer: https://github.com/sdslabs/rootex/blob/dev/docs/guides/getting_started.rst